interview-questions.md

```diff
@@ -9,47 +9,90 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
 Your answer:
+JSX is a hybrid of Javascript and HTML. The difference between them is JSX can use Javascript
+syntax and HTML syntax.
```

Comment:
This would be a great spot to insert an example such as onClick, className, etc.